gatsby-config.js

```diff
@@ -19,8 +19,8 @@ if ((metaConfig === null) || (metaConfig === undefined)) {
 
 module.exports = {
   adapter: adapter({
-    excludeDatastoreFromEngineFunction: false,
-    imageCDN: false,
+    excludeDatastoreFromEngineFunction: true,
+    imageCDN: true,
```

Comment:
```suggestion
    imageCDN: process.env.NETLIFY_IMAGE_CDN || true,
```

Reply:
Same comments for GATSBY_EXCLUDE_DATASTORE_FROM_BUNDLE apply here as well. If we find the NETLIFY_IMAGE_CDN variable automatically overrides the two default options, then there is no need for this change. Will test now.

Reply:
After testing this on a throwaway branch, the time didn't actually change (it was still 6 minutes, even when I set the variables to false). I'm going to remove this change request because the gatsby-adapter-netlify plugin seems to have taken both the variable and the option into account.

Reply:
Confirmed that the option value takes precedence, so recommending we allow the environment variable to take precedence if it exists (see suggestion above): https://github.com/gatsbyjs/gatsby/blob/c91ed287fd319a345c2f27877e20656826767e92/packages/gatsby-adapter-netlify/src/index.ts#L43